zookeeper-contrib/zookeeper-contrib-zkfuse/src/thread.h

```diff
@@ -41,7 +41,7 @@ class Thread {
 
     void Create(void* ctx, ThreadFunc func);
     void Join() {
-        //avoid SEGFAULT because of unitialized mThread
+        //avoid SEGFAULT because of initialized mThread
```

Comment:
uninitialized ?

In case of uninitialized, `_func` should be `NULL`.

https://github.com/apache/zookeeper/blob/master/zookeeper-contrib/zookeeper-contrib-zkfuse/src/thread.cc#L27-L39

Reply:
@kezhuw Thanks, fixed